### PR TITLE
[NIHONGO-0005] user service-Added put api

### PR DESCRIPTION
**Pull request by shoaib812**

### CODE REVIEW

File: src/main/java/com/jpn/lang/controller/EmployeeController.java [hunk 1]
```diff
@@ -58,4 +59,10 @@ public ResponseEntity<?> deleteEmployee(Long id) {
         employeeService.deleteEmployee(id);
         return ResponseEntity.ok("Delete...");
     }
+
+    @Override
+    public ResponseEntity<?> put(EmployeePutRequest putEmployee, Long id) {
```
Comment (Mohdzaved2434) [inline src/main/java/com/jpn/lang/controller/EmployeeController.java]: Can you return EmployeeResponse object from here not ResponseEntity<?>? 
same as post or get.

First convert Employee into EmployeeResponse 
then return.

we will use ResponseEntity<?>  later

File: pom.xml [hunk 2]
```diff
@@ -45,6 +45,16 @@
             <groupId>org.springframework.boot</groupId>
             <artifactId>spring-boot-starter-data-jpa</artifactId>
         </dependency>
+        <dependency>
```
Comment (Mohdzaved2434) [inline pom.xml]: We are not having swagger configuration in this pull request
Why did you add these dependencies in this pull request?

**Comment by Mohdzaved2434** [comment]
Did you also run sonarqube before pushing?

File: src/main/java/com/jpn/lang/controller/EmployeeController.java (hunk 1 above)
Comment (shoaib812) [reply src/main/java/com/jpn/lang/controller/EmployeeController.java]: I have completed this task..

File: pom.xml (hunk 2 above)
Comment (shoaib812) [reply pom.xml]: I have deleted swagger dependency.

File: pom.xml [hunk 3]
```diff
@@ -45,6 +45,11 @@
...
+            <groupId>org.apache.maven</groupId>
+            <artifactId>maven-model</artifactId>
+            <version>3.3.9</version>
```
Comment (Mohdzaved2434) [inline pom.xml:51]: what is the use of this dependency?

File: pom.xml (hunk 2 above)
Comment (Mohdzaved2434) [reply pom.xml]: <artifactId>maven-model</artifactId>
what is the use of this dependency?

Comment (shoaib812) [reply pom.xml]: i think, it is use in swagger .

Comment (Mohdzaved2434) [reply pom.xml]: But you have removed swagger dependency, why you did not remove it?  
And I dont think it is used in swagger

Comment (shoaib812) [reply pom.xml]: ****i have deleted the repo.****

**Review by Mohdzaved2434 — APPROVED** [review]
